### DISCUSSION TITLE
Dose configuration inside DICOM EGSPHANT file using synchronized simulation (SOURCE 21)

### DISCUSSION THREAD

**Discussion by physicsmed** [nrc-cnrc/EGSnrc#1136] [Questions] [answered]
Hello scientists
I am trying to extract dose values from 3ddose file in an EGSphant file (converted using ctcreate) using SOURCE 21 , but i am wondering how do i increase or i decrease the dose values, because MU Index is just a percentage, it must be from 0 to 1 in each simulation. so here is my questions:
1-How do I configure the dose in my simulation? is it related to number of histories?
2- What is the unit of dose in 3ddose file?
3- I am observing that the chair in my phantom also absorbs the dose. How do I avoid that?

**Comment by rtownson** [comment, marked as answer]
1. All quantities are reported `per source fluence` or `per source particle`, so that calculated quantities stay independent of the number of histories that you simulate. Since in reality you don't know how many source particles are needed to achieve a particular dose, you can use a calibration simulation for absolute dose conversion. Note that relative comparisons are always more reliable that absolute dose conversions.

https://pubmed.ncbi.nlm.nih.gov/16177516/

2. Gy per source particle
3. I'm not sure what you mean. Do you want to remove the CT chair from the phantom? You could crop the image (ctcreate has an option for this). To do something more complex, like replacing the chair structure with air, you'd have to write your own script.

**Reply by physicsmed** [reply]
Thank you for the information, what i mean in the last question is in my DICOM files including also a part of table where the patient set,  this region shows an absorbed dose. so my question is : is there any method to introduce the my Hounsfield numbers (HU) to ctcreate when converting from DICOM to egsphant? because i think i have problem of densities.

**Reply by rtownson** [reply]
What do you mean by "introduce the my Hounsfield numbers"? Using the CT ramp file when you run ctcreate, the HU values are converted to density and material types. You provide that conversion curve specifically to match your CT scanner calibration.

**Reply by physicsmed** [reply]
what i need is how to generate materials in egsphant same as CT scanner. For example let's say if we had HU between 226 and 3071, the ctcreate should give it BONE material. my question is simply, how to configure these intervals in order to ensure that materials in egsphant are generated properly, an match the CT scanner.

**Reply by rtownson** [reply]
This is done in the ctcreate input file, see section 16.2 of the DOSXYZ user manual. You provide the CT numbers (HU values) and the material and density range that corresponds. There is a fully worked example in the table on page 121.

**Reply by physicsmed** [reply]
Thank you Sir, i appreciate that. One more question is related to 3ddose file. Is there any way to extract DVH from 3ddose file?

**Reply by rtownson** [reply]
While some groups have coded this, they have not contributed it to EGSnrc. To do it yourself, you would need to write a script that reads an RD DICOM file and overwrites the dose values with those from the 3ddose file (after some interpolations). Then the RD DICOM file can be imported into your TPS. You can do this using python and pydicom. I think it is also possible using CERR in matlab.

**Reply by physicsmed** [reply]
Thank You, i will try to write it.

**Reply by rtownson** [reply]
3D slicer might be able to do it:
https://www.slicer.org/
https://discourse.slicer.org/t/loading-3ddose-files-dicom-image-in-3d-slicer/10082
